wemake_python_styleguide/visitors/ast/conditions.py

```diff
@@ -111,7 +118,12 @@ def _check_simplifiable_returning_if(self, node: ast.If) -> None:
                 if keywords.is_simple_return(else_body):
                     self.add_violation(SimplifiableReturningIfViolation(node))
                 return
-            body = getattr(node, 'wps_parent').body  # TODO: refactor
+
+            parent = getattr(node, 'wps_parent', None)
```

Comment:
This is not right for two reasons:
1. It uses `getattr` and not `get_parent`
2. It has `None` as the default value, but later it is used as `parent.body`. It will fail if `parent` is `None`. We don't check for this case